jupyterhub/templates/_helpers.tpl

```diff
@@ -172,12 +172,12 @@ component: {{ include "jupyterhub.componentLabel" . }}
 
     - https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
 */}}
-{{- define "jupyterhub.dockersingleuserconfigjson" -}}
-{{ include "jupyterhub.dockersingleuserconfigjson.yaml" . | b64enc }}
+{{- define "jupyterhub.dockerconfigjson" -}}
+{{ include "jupyterhub.dockerconfigjson.yaml" . | b64enc }}
 {{- end }}
```

Comment:
The chart previously allowed for the creation of separate secrets through the Helm chart, one for the hub pod, and one for the user pods, but now we only let them create a global or reference those they have manually created. This is what relates to the hard deprecation and `fail` command in the NOTES.txt file.